webrender_traits/src/webgl.rs

```diff
@@ -73,6 +73,7 @@ pub enum WebGLCommand {
     GetAttribLocation(WebGLProgramId, String, MsgSender<Option<i32>>),
     GetUniformLocation(WebGLProgramId, String, MsgSender<Option<i32>>),
     GetVertexAttrib(u32, u32, MsgSender<WebGLResult<WebGLParameter>>),
+    GetVertexAttribOffset(u32, u32, MsgSender<WebGLResult<isize>>),
```

Comment:
I used GetVertexAttribOffset because that's how the WebGL method is named: https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.10 WebGL methods generally match the ones in OpenGL ES, but not with this one.